data/authorities.json

```diff
@@ -109,30 +109,45 @@
         "name": "YW Electric Association"
       }
     },
-    "local": {
-      "co-city-and-county-of-denver": {
-        "name": "City and County of Denver"
-      },
+    "city": {
       "co-city-of-boulder": {
-        "name": "City of Boulder"
-      },
-      "co-boulder-county": {
-        "name": "Boulder County"
+        "name": "City of Boulder",
+        "city": "Boulder",
+        "county": "Boulder"
       },
       "co-town-of-avon": {
-        "name": "Town of Avon"
+        "name": "Town of Avon",
+        "city": "Avon",
+        "county": "Eagle"
       },
       "co-town-of-eagle": {
-        "name": "Town of Eagle"
+        "name": "Town of Eagle",
+        "city": "Eagle",
+        "county": "Eagle"
       },
       "co-town-of-erie": {
-        "name": "Town of Erie"
+        "name": "Town of Erie",
+        "city": "Erie",
+        "county": "Weld"
       },
       "co-town-of-vail": {
-        "name": "Town of Vail"
+        "name": "Town of Vail",
+        "city": "Vail",
+        "county": "Eagle"
+      }
+    },
+    "county": {
+      "co-city-and-county-of-denver": {
+        "name": "City and County of Denver",
+        "county": "Denver"
+      },
+      "co-boulder-county": {
+        "name": "Boulder County",
+        "county": "Boulder"
       },
       "co-unincorporated-eagle-county": {
-        "name": "Unincorporated Eagle County"
+        "name": "Unincorporated Eagle County",
```

Comment:
The language on the website is "Unincorporated Eagle County has specified bonuses and additional rebates for it’s [sic] residents and businesses that can be used in addition to Energy Smart rebates through Walking Mountains Science Center." That reflects at least some degree of agency/authority to me. From the Wikipedia article on unincorporated areas: "Most states have granted some form of home rule, so that county commissions (or boards or councils) have the same powers in these areas as city councils or town councils have in their respective incorporated areas."

https://www.walkingmountains.org/sustainability-hub/unincorporated-eagle-county-energy-efficiency-program/

So, while I agree that your proposed distinction could be helpful at some point, I think this is closer to a normal town/city authority than it sounds. This is one of a set of partnerships between Walking Mountains Science Center and towns/areas nearby (other examples include the towns of Vail and Aspen). The reality is that this is more of a "co-authority" partnership, but that would also require evolving the data model in a way that's probably not needed 99% of the time.